release/rancher/rancher.go

```diff
@@ -0,0 +1,80 @@
+package rancher
```

Comment:
This file needs to be formatted.

Reply:
I used go fmt to format it.

Reply:
I think you either need to check the config or use a different editor. This is formatted with 2 spaces. Go uses 4.